recipe-hooks-project/src/components/General.js

```diff
@@ -0,0 +1,170 @@
+import styled from "styled-components";
+import {Link} from "react-router-dom";
+
+const Wrapper = styled.div`
+  // height: 90vh;
+  padding: auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  color: blue;
+`;
+
+const Section = styled.div`
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  width: 90%;
+  padding-bottom: 1.5em;
+  margin: auto;
+`;
+
+const Title = styled(Section)`
+  height: 1em;
+  font-size: 1.5em;
+  font-weight: 500;
+  margin: auto;
+  justify-content: center;
+  padding-top: 1.5em;
+`;
+
+const OverlayOpaqueBackground = styled.div.attrs((props) => ({
+  ...props,
+}))`
+  z-index: 10;
+  width: 25%;
+  height: 100px;
+  position: absolute;
+  left: 0; 
+  top: 15%; 
+  right: 0; 
+  bottom: 0;
+  margin: 0 auto;
+  padding: 0;
+  background: white;
+  
+  display: ${props => props.show ? "" : "none"};
+`;
+
+const Overlay = styled(OverlayOpaqueBackground).attrs((props) => ({
+  ...props,
+}))`
+  color: red;
+  background: white;
+  font-weight: 400;
+  border: 2px solid red;
+  -webkit-box-sizing: border-box;
```

Comment:
`styled-components` will automatically apply autoprefixer to your styles, no need to prefix them on your own